compiler/back_end/util/code_template.py

```diff
@@ -22,59 +22,7 @@
 import string
 
 
-class _CppFormatter(string.Formatter):
-  """Customized Formatter using $_name_$ instead of {name}.
```

Comment:
I'm unclear on why `$_name_$` is used instead of `{name}` but it does mean that the templates might be syntactically valid C++. This makes it easier to edit the templates with syntax highlighting in an IDE. Maybe there's some way to get that back in the future but nothing comes to mind.

Reply:
I've been editing with VSCode with these changes, syntax highlighting is okay. Maybe this is an issue for specific IDEs.

![image](https://github.com/google/emboss/assets/426919/40483620-131a-402f-8164-e1c218fa515e)